src/components/PageTitle.jsx

```diff
@@ -0,0 +1,11 @@
+import Head from 'next/head';
+
+export default function PageTitle({ title }) {
+  return (
+    <Head>
+      <title>{ title } | Accessible Components</title>
+      <meta name="description" content="Sparkbox accessible components" />
+      <link rel="icon" href="/favicon.png" />
+    </Head>
```

Comment:
What do you think about refactoring this to be a `<Layout>` component? We can keep the `<Head>` piece as-is, but we could include the `<Header>` and render `{children}` inside a `main` tag as well. Not critical, but something that will help with re-usability.